src/components/NavBar.vue

```diff
@@ -5,7 +5,7 @@
     fluid
   >
     <v-app-bar
-      app
+      app1
```

Comment:
@vickywane what is the purpose of renaming app to app1?

Reply:
Was from a previous change. I have restored it back to `app`